plugin.xml

```diff
@@ -75,59 +75,31 @@
-    <header-file src="src/ios/headers/SFUserAccount.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFUserAccountConstants.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFUserAccountManager.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFUserAccountIdentity.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SalesforceSDKManager.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SalesforceSDKManagerWithSmartStore.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFAuthErrorHandler.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFAuthErrorHandlerList.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SalesforceSDKCoreDefines.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFLoginViewController.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFSDKLoginHostDelegate.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFSDKLoginHost.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFSDKLoginHostListViewController.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/SFSDKLoginHostStorage.h" target-dir="salesforce" />
-    <header-file src="src/ios/headers/AppDelegate+SalesforceHybridSDK.h"  />
-    <header-file src="src/ios/headers/UIApplication+SalesforceHybridSDK.h"  />
-
     <!-- Source files -->
     <source-file src="src/ios/classes/AppDelegate+SalesforceHybridSDK.m" />
     <source-file src="src/ios/classes/UIApplication+SalesforceHybridSDK.m" />
     <source-file src="src/ios/classes/InitialViewController.m" />
 
+    <!-- Header files -->
+    <header-file src="src/ios/classes/AppDelegate+SalesforceHybridSDK.h" />
+    <header-file src="src/ios/classes/UIApplication+SalesforceHybridSDK.h" />
+    <header-file src="src/ios/classes/InitialViewController.h" />
+
     <!-- Resource bundles -->
     <resource-file src="src/ios/resources/SalesforceSDKResources.bundle" />
     <resource-file src="src/ios/resources/Images.xcassets" />
     <resource-file src="src/ios/resources/SalesforceSDKAssets.xcassets" />
 
-    <!-- Libraries -->
-    <source-file src="src/ios/frameworks/libSalesforceAnalytics.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSalesforceSDKCore.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSmartStore.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSmartSync.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSalesforceHybridSDK.a" framework="true" />
+    <!-- Static binaries -->
     <source-file src="src/ios/frameworks/libsqlcipher.a" framework="true" />
-    <source-file src="src/ios/frameworks/libCocoaLumberjack.a" framework="true" />
+
+    <!-- Dynamic framework binaries -->
+    <framework src="src/ios/frameworks/SalesforceAnalytics.framework" custom="true" embed="true" />
```

Comment:
Adding frameworks instead of static binaries. The `embed` flag tells `Cordova` to embed our frameworks in the generated app.